tutorials/fundamentals/async-foreach.md

```diff
@@ -0,0 +1,15 @@
+You should **not** make the callback function parameter in `forEach()`
+an async function as you cannot guarantee order of iteration.
```

Comment:
Not exactly true. The biggest problem is that, if you pass an async function as a `forEach()` callback, there's no way for you to handle errors.

The other surprise is that `forEach()` executes async callbacks in parallel rather than in series, which is what the example shows.